packages/neuron-ui/src/widgets/CopyZone/index.tsx

```diff
@@ -15,14 +15,18 @@ const CopyZone = ({ children, content, name, style, className = '' }: CopyZonePr
   const timer = useRef<ReturnType<typeof setTimeout>>()
   const prompt = copied ? t('common.copied') : name || t(`common.copy`)
 
-  const onCopy = useCallback(() => {
-    setCopied(true)
-    window.navigator.clipboard.writeText(content)
-    clearTimeout(timer.current!)
-    timer.current = setTimeout(() => {
-      setCopied(false)
-    }, 1000)
-  }, [setCopied, content])
+  const onCopy = useCallback(
+    (event: React.MouseEvent<HTMLDivElement>) => {
+      event.stopPropagation()
```

Comment:
Try to avoid using stopPropagation as much as possible, as it is not a good pattern. Related articles:
https://stackoverflow.com/a/54817255/21858805

Reply:
@WhiteMinds 
I tried to separate the 'copy' operation from the click event by using the type of event.target to determine whether the user clicked 'copy' or 'expand'. However, in order to do this, the information about which row was clicked needs to be passed into the Table component through props for it to determine whether to expand. This would greatly increase the complexity of the Table component, making it messy. It would also add a burden to the render function, and I believe that determining which row was clicked should indeed be maintained by the Table component itself. Therefore, I choose to continue using stopPropagation to prevent its bubbling behavior to achieve our needs.

Reply:
The drawbacks of using stopPropagation are discussed in the link provided above, but to summarize, it can cause issues such as disabling onClickOutside.

If this decision is made, it is likely that the bad pattern will spread further when other clickable elements are added to the rows in the future, and it will be difficult to reverse.

I can suggest some alternative solutions, but they will all require event checking in onRowClick of the table to determine whether to execute the expand function:

1. The most common solution is to check event.target. If this is deemed too complex or interferes with readability, we can use other solutions.
If we want to use this approach, we can further consider how to design it.

3. Add a new property to the event, such as preventTableExtend (of type symbol, to prevent interference from other components that may add this property).
Then, the Table component can export two functions: setPreventTableExtend(event, bool) and isPreventTableExtend(event).
This will limit this special pattern to the Table component and all places that use this function will know exactly what they are doing, unlike event.stopPropagation(), which makes it difficult to remove because developers are unsure which component it is serving.

4. Use the built-in [cancelable](https://developer.mozilla.org/en-US/docs/Web/API/Event/cancelable) property, but this is not common and can reduce readability, so it is not recommended.

Reply:
@WhiteMinds update